marlowe-actus/src/Language/Marlowe/ACTUS/Model/APPLICABILITY/ApplicabilityModel.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
+{-# OPTIONS_GHC -fno-warn-name-shadowing #-}
+
+module Language.Marlowe.ACTUS.Model.APPLICABILITY.ApplicabilityModel where
+
+import Data.Maybe ()
+import Language.Marlowe.ACTUS.Definitions.ContractTerms (ContractTerms(..), TermValidationError(..))
+import Data.Validation (Validation(..))
+
+-- Applicability Validation Functions
```

Comment:
These applicability functions are defined in the way that I felt best matched with the specification. The functions represent constraints used in the specification. In the Applicability.hs file, we chain these constraints for each contract based on the spec for the contract.